tpl/collections/collections_test.go

```diff
@@ -811,7 +883,7 @@ func BenchmarkD2(b *testing.B) {
 		name := fmt.Sprintf("n=%d,max=%d", n, max)
 		b.Run(name, func(b *testing.B) {
 			for b.Loop() {
-				ns.D(seed, n, max)
+				_, _ = ns.D(uint64(seed), uint64(n), uint64(max))
```

Comment:
Not much practical difference, but it looks odd, so I think we might as well just change the signature of runBenchmark ...